plotly/tools.py

```diff
@@ -2275,6 +2277,25 @@ def create_candlestick(open, high, low, close,
         return dict(data=data, layout=layout)
 
 
+    @staticmethod
+    def create_dendrogram(X, orientation="bottom", labels=None,
+                          colorscale=None, **kwargs):
+        """
+        Returns a dendrogram Plotly figure object.
+
+        X: Heatmap matrix as array of arrays
```

Comment:
In this case it's to distinguish between a linkage matrix (Z) and the initial array of observations(X), like in http://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.distance.pdist.html#scipy.spatial.distance.pdist